idd/Energy+.idd.in

```diff
@@ -37014,10 +37014,16 @@ ZoneHVAC:EvaporativeCoolerUnit,
         \note optional, used for direct/indirect configurations
         \type object-list
         \object-list EvapCoolerNames
-   A15; \field Design Specification ZoneHVAC Sizing Object Name
+   A15, \field Design Specification ZoneHVAC Sizing Object Name
         \note Enter the name of a DesignSpecificationZoneHVACSizing object.
         \type object-list
         \object-list DesignSpecificationZoneHVACSizingName
+   N4;  \field Shut Off Relative Humidity
```

Comment:
This is a new optional field, so I assume the code below will check for blank.  I see in the IDD we sometimes have "Relative Humidity Fraction" from 0 to 1, but we also have it as a percent from 0 to 100.  So this is fine.